frontend/micro-ui/web/micro-ui-internals/packages/modules/fs/src/Module.js

```diff
@@ -4,23 +4,11 @@ import { useTranslation } from "react-i18next";
 import { useRouteMatch } from "react-router-dom";
 import CitizenApp from "./pages/citizen";
 import Create from "./pages/citizen/create/index";
-import EmployeeApp from "./pages/employee";
-import BrSelectName from "./pagecomponents/BrSelectName";
-import BRSelectPhoneNumber from "./pagecomponents/BrSelectPhoneNumber";
-import BRSelectGender from "./pagecomponents/BRSelectGender";
```

Comment:
these files doesn't exists but it was added.
First create a module with one module file.